src/js/components/streaming-api-handler.tsx

```diff
@@ -372,10 +386,6 @@ class StreamingAPIHandler extends React.Component<Props, void> {
   }
 }
 
-const mapStateToProps = (state: StateTree): GeneratedStateProps => ({
-  team: User.selectors.getTeam(state),
-});
-
 const mapDispatchToProps = (dispatch: Dispatch<any>): GeneratedDispatchProps => ({
```

Comment:
Actually, looking at the type definition, the type seems to be the type of the state tree, not actions. But it's not used anywhere, so it doesn't seem to have any effect:
```
export interface Dispatch<S> {
    <A extends Action>(action: A): A;
}
```